src/app/containers/basket.component.ts

```diff
@@ -30,7 +38,25 @@ export class BasketController {
   }
 
   makeOrder(): void {
-    this.lOrderService.makeOrder(this.order);
+    this.lOrderService.makeOrder(this.order)
+      .then(res => {
+        this.showToast('Order has been placed!');
+      })
+      .catch(err => {
+        this.showToast('Error! Unable to place order');
+      })
+      .finally(() => {
+        this.$state.go('week-menu');
+      });
+  }
+
+  private showToast(msg: string): void {
```

Comment:
I'm using built-in service for toasts from Angular Material. It is already implement. `showToast` is just a helper method for this particular UI component.